cc/tf_bt_utils.cc

```diff
@@ -127,6 +128,44 @@ void WriteGameExamples(const std::string& gcp_project_name,
             << examples.size() << std::endl;
 }
 
+void WriteEvalRecord(const std::string& gcp_project_name,
+                     const std::string& instance_name,
+                     const std::string& table_name,
+                     const MctsPlayer& player,
+                     const std::string& black_player_name,
+                     const std::string& white_player_name) {
+  Table table(CreateDefaultDataClient(gcp_project_name, instance_name,
+                                      google::cloud::bigtable::ClientOptions()),
+              table_name);
+
+  // Retrieve the game counter from the Bigtable and increment it atomically.
+  using namespace google::cloud::bigtable;
+  auto rule =
+      ReadModifyWriteRule::IncrementAmount("metadata", "game_counter", 1);
```

Comment:
This increments the main game counter. I'm should probably change this eval_counter